ztrouy/frontend-capstone-api
# Feature - Get Users

### Purpose
To allow Users to see other Users

### Files Changed
- Added serializers,  along with list and retrieve views in `users/py`
- Added routing for Users View Set in `urls.py`

### To Test
Fetch, setup by running `pipenv shell` and `./seed_database.sh` in the project root directory, run the debugger, then confirm the following
- GET endpoint `/users` returns a list of all Users
- GET endpoint `/users/{id}` returns the appropriate User if exists, along with their Groups, Games, and if they are the same as the User requesting them

closes #10
